assets/tvs/multitv/js/multitv.js

```diff
@@ -1013,22 +1024,24 @@
         },
 
         // save/append edit box
-        saveRow: function (mode) {
+        saveRow: function (mode, values) {
             var _this = this;
 
             if (typeof tinyMCE !== 'undefined') {
                 tinyMCE.triggerSave();
             }
-            var values = {};
-            var saveTab = $('[name^="' + _this.tvid + 'tab_radio_mtv"]', _this.fieldEditForm).getValue();
-            values.fieldTab = (saveTab !== '') ? saveTab : '';
-            $.each(_this.fieldNames, function () {
-                var fieldInput = $('[name^="' + _this.tvid + this + '_mtv"][type!="hidden"]', _this.fieldEditForm);
-                values[this] = fieldInput.getValue();
-                if (fieldInput.hasClass('mtvImage')) {
-                    _this.setThumbnail(values[this], fieldInput.attr('name'), _this.fieldEditForm);
-                }
-            });
+            if(values === false) {
```

Comment:
I would prefer undefined over false: `(typeof values === "undefined")` . It occurs automatic if the function call gets fewer arguments than it was declared with.